db/test/fns/queue_artifacts_test.js

```diff
@@ -456,6 +456,67 @@ suite(testing.suiteName(), function() {
     assert.deepEqual(rows.map(({ run_id }) => run_id), [0, 2, 4, 6, 8]);
   });
 
+  helper.dbTest('get_expired_artifacts_for_deletion', async function(db) {
+    const yesterday = taskcluster.fromNow('-1 day');
+    const tomorrow = taskcluster.fromNow('1 day');
+    const today = new Date();
+    const taskId = slugid.nice();
+    for (let i = 0; i < 10; i++) {
+      await db.fns.create_queue_artifact(
+        taskId,
+        i,
+        `name-${i}`,
+        'storage-type',
+        'content-type',
+        {},
+        false,
+        (i & 1) ? tomorrow : yesterday,
+      );
+    }
+
+    let rows = await db.fns.get_expired_artifacts_for_deletion({
+      expires_in: today,
+      page_size_in: 1000,
+    });
+    assert.deepEqual(rows.map(({ run_id }) => run_id), [0, 2, 4, 6, 8]);
+  });
+
+  helper.dbTest('delete_queue_artifacts', async function(db) {
```

Comment:
Since the `get_expired_artifacts_for_deletion` test is the exact same as the first lines of this `delete_queue_artifacts` test, I think it's safe to just keep this second test and wipe out the first.

```suggestion
  helper.dbTest('get_expired_artifacts_for_deletion and delete_queue_artifacts', async function(db) {
```

Reply:
Yeah, I think you're right.. I thought that maybe at some point they will diverge, so having individual tests is better, but until this happens we can bundle them :+1: